src/main/java/edu/greenblitz/robotName/subsystems/arm/ElbowUtils/FalconElbow/FalconElbowConstants.java

```diff
@@ -52,13 +64,30 @@ public class FalconElbowConstants {
         SOFTWARE_LIMIT_SWITCH_CONFIGS.ReverseSoftLimitThreshold = BACKWARD_ANGLE_LIMIT.getRadians();
     }
 
+    public static final HardwareLimitSwitchConfigs HARDWARE_LIMIT_SWITCH_CONFIGS = new HardwareLimitSwitchConfigs();
+    static{
+        HARDWARE_LIMIT_SWITCH_CONFIGS.ForwardLimitAutosetPositionEnable = true;
+        HARDWARE_LIMIT_SWITCH_CONFIGS.ForwardLimitAutosetPositionValue = FORWARD_ANGLE_LIMIT.getRadians();
```

Comment:
i think its in rotations. change to .getRotations(). and this is out of cr scope so next time open a new branch for it.